tests/drivers/abp2/setenv.venv

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+export BOARD=nucleo-l031k6
+export PROGRAMMER=cpy2remed
+alias make="CFLAGS+='-Wno-error=unused-variable -Wno-error=unused-const-variable \
+	-Wno-error=unused-but-set-variable -Wno-error=unused-parameter' \
+	make"
```

Comment:
What do you need this for?

Reply:
It's a file from my own needs. I didn't read `.gitignore` carefully enough... I'll remove it.